docs/api/rest/config.md

```diff
@@ -209,117 +149,143 @@ curl --location --request GET 'http://localhost:5000/api/config/sourceaetitle'
 
 ---
 
-## GET /api/config/destinationaetitle
+## GET /api/config/sourceaetitle/{ae-title}
 
-Retrieves a list of Destination AE Titles.
+Retrieves the named source AE Title.
```

Comment:
Meant to say AE attributes for the named AET? otherwise this reads like passing in AET to get AET, as it led me on first glance.

Reply:
Fixed